WalletWasabi/Backend/Models/FilterModel.cs

```diff
@@ -17,7 +17,7 @@ public class FilterModel
 		[JsonConverter(typeof(HeightJsonConverter))]
 		public Height BlockHeight { get; set; }
 
-		[JsonConverter(typeof(Uint256JsonConverter))]
+		[JsonConverter(typeof(UInt256JsonConverter))]
```

Comment:
The type that is converted is called `Uint256` and for that reason I think the converter should be called `Uint256Converter`.

Reply:
Forget it, the change is correct.

Reply:
No, it's not, the change is the Java naming, the original is the C# naming pattern.

Reply:
I will fix that.

Reply:
@nopara73 Changed it back to `Uint256JsonConverter`.